src/jqLite.js

```diff
@@ -638,33 +638,32 @@ forEach({
   },
 
   attr: function(element, name, value) {
+    var ret;
```

Comment:
I prefer small functions so that there is not much difference between the two approaches :-P